src/olympia/users/management/commands/process_users.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
+
+from django.db.models import Q
+
+from olympia.amo.management import ProcessObjectsCommand
+from olympia.users.models import UserProfile
+from olympia.users.tasks import (
+    backfill_activity_and_iplog,
+)
+
+
+class Command(ProcessObjectsCommand):
+    def get_model(self):
+        return UserProfile
+
+    def get_tasks(self):
+        return {
+            # Don't forget to use the --with_deleted argument when triggering
+            # this task!
+            'backfill_activity_and_iplog': {
+                'task': backfill_activity_and_iplog,
+                'queryset_filters': [
+                    ~Q(last_login_ip='') & Q(last_login__gte=datetime(2019, 1, 1))
```

Comment:
I thought we cleared the logs after 90 days (increasing to 180 days) so would there be anything after `datetime(2022, 5, 1)`?  (Or, if there will be, would those logs have been deleted long ago if this code was in place in 2019?)

Reply:
We're keeping `LOG_IN` (and `RESTRICTED`) activities forever. The 90/180 days is for `ADD_RATING` (and a bunch of others that existed before) which I'm not backfilling (see https://github.com/mozilla/addons-server/issues/19602#issuecomment-1227360838)